source/_patterns/01-elements/loading-indicator/00-loading-indicator.hbs

```diff
@@ -0,0 +1,15 @@
+<svg
+	class="elm-loadingindicator elm-loadingindicator--size-{{ size }}"
```

Comment:
@annsch let's use the special aspect of the `size` as a `data-size` attribute (adapted from e.g. buttons).